board.cpp

```diff
@@ -0,0 +1,118 @@
+#include <fstream>
+#include <random>
+#include <iostream>
+#include "Board.h"
+
+std::default_random_engine generator;
+std::uniform_int_distribution<int> distribution(1, 6);
+
+Board::Board(std::string inputFilePath, std::string outputFilePath)
+{
+    in_path.open(inputFilePath);
```

Comment:
should not be part of board. all objects should be passed to board. input should not be taken in board class

Reply:
So, input should be in the main driver function, where I will initialize all the objects (players, snakes etc.)?

Reply:
in the driver class itself

Reply:
Okay, thanks a lot for the help!